ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -69,9 +76,22 @@ const validators = {
   areaOfResearch: required,
 };
 
+interface ProfilePageProps {
+  profileState: {
```

Comment:
I think this is like this because we lump them together in the `userProfileStore`. I don't know enough about how we do global state to know how to disentangle that on the top of my head. I didn't create these - just pulled them out of inline props / state.

Reply:
Ah – I didn't really notice this was just adding types to existing code. Feel free to punt on this to whatever extent you'd like. It would probably be a nice cleanup, but I don't want to create huge amounts of unnecessary effort.